build/UIKit/lib/UIKitLib.vcxproj

```diff
@@ -143,7 +143,9 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\UIKit\UICollectionViewCell.mm">
       <ObjectiveCARC>true</ObjectiveCARC>
     </ClangCompile>
-    <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\UIKit\UIGridLayoutSection.mm" />
+    <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\UIKit\UIGridLayoutSection.mm">
```

Comment:
>IGridLayoutSection.mm" [](start = 81, length = 22)

Mind adding #import "AssertARCEnabled.h" to the tops of these files?  It ensures we never accidentally turn ARC back off...